dali/operators/reader/loader/nemo_asr_loader.h

```diff
@@ -89,7 +95,8 @@ namespace detail {
 
 DLL_PUBLIC void ParseManifest(std::vector<NemoAsrEntry> &entries, std::istream &manifest_file,
                               double min_duration = kDefaultDuration,
-                              double max_duration = kDefaultDuration);
+                              double max_duration = kDefaultDuration,
+                              bool read_text = true);
```

Comment:
I know that reading whole reader makes the `read_text` meaning obvious, but when somebody reviews only a part of it, `read_text` looks ambiguous. Could you add a doc to this function?

Reply:
Done